tools/reckless

```diff
@@ -0,0 +1,684 @@
+                                          warn=warn)
+
+
+class RecklessConfig(Config):
+    """Reckless config (by default, specific to the bitcoin network only.)
+    This is inherited by the lightningd config and contains all reckless
+    maintained plugins."""
+
+    def enable_plugin(self, plugin_path):
+        """Handle persistent plugin loading via config"""
+        self.editConfigFile(f'plugin={plugin_path}',
+                            f'disable-plugin={plugin_path}')
+
+    def disable_plugin(self, plugin_path):
+        """Handle persistent plugin disabling via config"""
+        self.editConfigFile(f'disable-plugin={plugin_path}',
+                            f'plugin={plugin_path}')
+
+    def __init__(self, path=None, default_text=None):
+        if path is None:
+            path = os.path.join(LIGHTNING_DIR, 'reckless',
+                                'bitcoin-reckless.conf')
+        if default_text is None:
+            default_text = '# This configuration file is managed by reckles' +\
+                           's to activate and disable\n# reckless-installed' +\
+                           ' plugins\n\n'
+        Config.__init__(self, path=str(path), default_text=default_text)
+        self.reckless_dir = os.path.split(path)[0]
+
+
+class LightningBitcoinConfig(Config):
+    """lightningd config specific to the bitcoin network. This is inherited by
+    the main lightningd config and in turn, inherits bitcoin-reckless.conf."""
+
+    def __init__(self, path=None, default_text=None, warn=True):
+        if path is None:
+            path = os.path.join(LIGHTNING_DIR, 'bitcoin', 'config')
+        if default_text is None:
+            default_text = "# This config was autopopulated by reckless\n\n"
+        Config.__init__(self, path=str(path),
+                        default_text=default_text, warn=warn)
+
+
+class InferInstall():
+    """Once a plugin is installed, we may need its directory and entrypoint"""
+    def __init__(self, name):
+        reck_contents = os.listdir(RECKLESS_CONFIG.reckless_dir)
+        if name[-3:] == '.py':
+            name = name[:-3]
+        if name in reck_contents:
+            self.dir = os.path.join(RECKLESS_CONFIG.reckless_dir, name)
+        else:
+            raise Exception(f"Could not find a reckless directory for {name}")
+        plug_contents = os.listdir(os.path.join(RECKLESS_CONFIG.reckless_dir,
+                                                name))
+        for n in py_entry_guesses(name):
+            if n in plug_contents:
+                self.entry = os.path.join(self.dir, n)
+                self.name = n
+                return
+        raise Exception(f'plugin entrypoint not found in {self.dir}')
+
+
+def help(target):
+    if len(target) > 0:
+        print(globals()[target[0]].__doc__)
+    else:
+        parser.print_help(sys.stdout)
+
+
+def verbose(*args):
+    if not IS_VERBOSE:
+        return
+    print(*args)
```

Comment:
Python has quite the powerful `logging` library at your disposal: https://docs.python.org/3/library/logging.html